state/sqlite/sqlite_metadata.go

```diff
@@ -21,34 +21,38 @@ import (
 
 	"github.com/dapr/components-contrib/metadata"
 	"github.com/dapr/components-contrib/state"
-	"github.com/dapr/kit/ptr"
 )
 
 const (
-	defaultTableName       = "state"
-	defaultCleanupInternal = 3600 // In seconds = 1 hour
-	defaultTimeout         = 20   // Default timeout for database requests, in seconds
+	defaultTableName         = "state"
+	defaultMetadataTableName = "metadata"
+	defaultCleanupInternal   = time.Duration(0) // Disabled by default
+	defaultTimeout           = 20 * time.Second // Default timeout for database requests, in seconds
+	defaultBusyTimeout       = 2 * time.Second
 
 	errMissingConnectionString = "missing connection string"
 	errInvalidIdentifier       = "invalid identifier: %s" // specify identifier type, e.g. "table name"
 )
 
 type sqliteMetadataStruct struct {
-	ConnectionString         string `json:"connectionString" mapstructure:"connectionString"`
-	TableName                string `json:"tableName" mapstructure:"tableName"`
-	TimeoutInSeconds         string `json:"timeoutInSeconds" mapstructure:"timeoutInSeconds"`
+	ConnectionString   string `json:"connectionString" mapstructure:"connectionString"`
+	TableName          string `json:"tableName" mapstructure:"tableName"`
+	MetadataTableName  string `json:"metadataTableName" mapstructure:"metadataTableName"`
+	TimeoutInSeconds   string `json:"timeoutInSeconds" mapstructure:"timeoutInSeconds"`
+	CleanupIntervalStr string `json:"cleanupInterval" mapstructure:"cleanupInterval"` // Cleanup interval as a time.Duration string
```

Comment:
Just make this `time.Duration` or if preferred `*time.Duration` directly -- our `metadata.DecodeMetadata` function supports this out of the box.

Reply:
Done